src/main/java/org/cactoos/func/FuncWithFallback.java

```diff
@@ -95,20 +95,21 @@
     /**
      * The func.
      */
-    private final Func<X, Y> func;
+    private final Func<? super X, ? extends Y> func;
 
     /**
      * The fallbacks.
      */
-    private final Iterable<Fallback<Y>> fallbacks;
+    private final Iterable<Fallback<? extends Y>> fallbacks;
```

Comment:
@DmitryBarskov since `Fallback` is an interface, it can be subtyped, so we should have `Iterable<? extends Fallback<...>>`